tables/superdarn/hdw/hdw.dat.sys

```diff
@@ -6,11 +6,10 @@
 #
 # **********************************************************************
 #
-#  1   2      3        4         5       6      7      8    9   10   11   12  13(1)  13(2)  13(3)  14   15  16 17
-#
-# UNTIL 01/Jan/1999 00:00:00.0
-   12 1999        0  -69.000   39.580     0.0  159.0 -3.33  1   10  0.000  1   -5.0   92.0    0.8  50.0  3  75 16
-#
-# UNTIL 01/Jan/3000 05:45:36.0
-   12 2999 31556736  -69.000   39.580     0.0  165.0 -3.33  1   10  0.000  1   -5.0   92.0    0.8  50.0  3  75 16
+#                            GEOGRAPHIC             BORE-   OFF  BEAM V  |--------- interferometer ---------| |--- RX ---| RNG MX
+#      YYYYMMDD HH:MM:SS    LAT      LON      ALT   SIGHT   SET  SEP  SG PS  TDIFF  TDIFF    X      Y     Z    RXRIS  ATTN     BM
+ 12  1 19950201 00:00:00  -69.000   39.580     0.0  159.0  0.00 -3.33 -1  1  0.000  0.000   -5.0   92.0   0.8   50.0  10 3  75 16
+ 12  1 19960424 11:00:00  -69.000   39.580     0.0  159.0  0.00 -3.33  1  1  0.000  0.000   -5.0   92.0   0.8   50.0  10 3  75 16
+ 12  1 19990101 00:00:00  -69.000   39.580     0.0  159.0  6.00 -3.33  1  1  0.000  0.000   -5.0   92.0   0.8   50.0  10 3  75 16
+ 12  1 20001231 06:00:00  -69.000   39.580     0.0  159.0  0.00 -3.33  1  1  0.000  0.000   -5.0   92.0   0.8   50.0  10 3  75 16
```

Comment:
Confirming the updates are intended.

Reply:
Good catch @aburrell - I forgot that these changes were implemented.  I reached out to Sessai earlier in the summer and am still waiting on a response, but based on the information from these pages I feel more confident in the revised versions on this branch than the current hardware files for Syowa South and East:

http://polaris.nipr.ac.jp/~SD/sensu/SW/IQreverse/
https://web.archive.org/web/20070719171647/http://www.uap.nipr.ac.jp/SD/SyowaSouthNew/